pythonforandroid/recipes/audiostream/__init__.py

```diff
@@ -18,12 +18,14 @@ def get_recipe_env(self, arch):
             sdl_include = 'SDL2'
             sdl_mixer_include = 'SDL2_mixer'
             env['USE_SDL2'] = 'True'
-            env['SDL2_INCLUDE_DIR'] = '/home/kivy/.buildozer/android/platform/android-ndk-r9c/sources/android/support/include'
+            env['SDL2_INCLUDE_DIR'] = join(self.ctx.bootstrap.build_dir, 'jni', 'SDL', 'include')
 
         env['CFLAGS'] += ' -I{jni_path}/{sdl_include}/include -I{jni_path}/{sdl_mixer_include}'.format(
                               jni_path=join(self.ctx.bootstrap.build_dir, 'jni'),
                               sdl_include=sdl_include,
                               sdl_mixer_include=sdl_mixer_include)
+        env['NDKPLATFORM'] = self.ctx.ndk_platform
+        env['LIBLINK'] = 'NOTNONE'
```

Comment:
Can you tell us about that line?

Reply:
Audiostream setup.py checks for env variable 'LIBLINK' in order to set android as build target.